doc/src/fix_qtpie_reaxff.rst

```diff
@@ -0,0 +1,196 @@
+:doc:`pair_style reaxff <pair_reaxff>` command and the ReaxFF force field
+file it reads in.  If a file name is specified for *params*, then the
+parameters are taken from the specified file and the file must contain
+one line for each atom type.  The latter form must be used when performing
+QTPIE with a non-ReaxFF potential. Each line should be formatted as follows,
+ensuring that the parameters are given in units of eV, eV, and :math:`\AA^{-1}`,
+respectively:
+
+.. parsed-literal::
+
+   itype chi eta gamma
+
+where *itype* is the atom type from 1 to Ntypes. Note that eta is
+defined here as twice the eta value in the ReaxFF file.
+
+The overlap integrals in the equation for :math:`\chi_{\mathrm{eff},i}`
+are computed by using normalized 1s Gaussian type orbitals. The Gaussian
+orbital exponents, :math:`\alpha`, that are needed to compute the overlap
+integrals are taken from the file given by *gfile*.
+This file must contain one line for each atom type and provide the Gaussian
+orbital exponent for each atom type in units of inverse square Bohr radius.
+Each line should be formatted as follows:
+
+.. parsed-literal::
+
+   itype alpha
+
+Empty lines or any text following the pound sign (#) are ignored. An example
+*gfile* for a system with two atom types is
+
+.. parsed-literal::
+
+    # An example gfile. Exponents are taken from Table 2.2 of Chen, J. (2009).
+    # Theory and applications of fluctuating-charge models.
+    # The units of the exponents are 1 / (Bohr radius)^2 .
+    1  0.2240  # O
+    2  0.5434  # H
+
+The optional *maxiter* keyword allows changing the max number
+of iterations in the linear solver. The default value is 200.
+
+.. note::
+
+   In order to solve the self-consistent equations for electronegativity
+   equalization, LAMMPS imposes the additional constraint that all the
+   charges in the fix group must add up to zero.  The initial charge
+   assignments should also satisfy this constraint.  LAMMPS will print a
+   warning if that is not the case.
+
+Restart, fix_modify, output, run start/stop, minimize info
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+No information about this fix is written to :doc:`binary restart files
+<restart>`.  This fix computes a global scalar (the number of
+iterations) and a per-atom vector (the effective electronegativity), which
+can be accessed by various :doc:`output commands <Howto_output>`.
+No parameter of this fix can be used with the *start/stop* keywords of
+the :doc:`run <run>` command.
+
+This fix is invoked during :doc:`energy minimization <minimize>`.
+
+Restrictions
+""""""""""""
+
+This fix is part of the REAXFF package.  It is only enabled if
+LAMMPS was built with that package. See the :doc:`Build package
+<Build_package>` page for more info.
+
+This fix does not correctly handle interactions involving multiple
+periodic images of the same atom.  Hence, it should not be used for
+periodic cell dimensions less than 10 Angstroms.
```

Comment:
Wouldn't this depend on the cutoff? Could say something like "Hence, it should not be used for
periodic cell dimensions less than the pairwise cutoff, typically 10 Angstroms."

Reply:
Good point. I have updated the documentation (I have also added this change to the documentation of fix qeq/reaxff and fix acks2/reaxff).